components/style/properties/helpers/animated_properties.mako.rs

```diff
@@ -2817,6 +2929,16 @@ impl<LengthType> Animatable for SVGStrokeDashArray<LengthType>
             }
         }
     }
+
+    #[inline]
+    fn add(&self, _other: &Self) -> Result<Self, ()> {
+        Err(())
```

Comment:
I wonder if this deserves a comment saying that SVG explicitly says that stroke-dasharray is non-additive.[1]

[1] https://www.w3.org/TR/SVG11/painting.html#StrokeDasharrayProperty